solutions/usaco-688.mdx

```diff
@@ -57,4 +57,73 @@ for j in pers:
 print(maxwins)
 ```
 </PySection>
+
+<JavaSection>
+```java
+
+import java.io.*;
+import java.util.*;
+
+public class HPS {
+	public static void main(String[] args) throws Exception {
+
+		
+	    Scanner in = new Scanner(new File("hps.in"));
```

Comment:
don't use scanner

Reply:
yeah, got it, i will change it. This is from a long time ago, and I was just testing it out. Thanks